src/sbvr-api/sbvr-utils.coffee

```diff
@@ -662,6 +661,13 @@ exports.getAbstractSqlModel = getAbstractSqlModel = (request) ->
 	request.abstractSqlModel ?= abstractSqlModels[request.vocabulary]
 	return request.abstractSqlModel
 
+cloneReq = (req, odata) ->
+	req.method = odata.method
+	req.url = odata.url
+	# set values back on odata body
+	req.body = odata.values
+	_.cloneDeep(req)
```

Comment:
`cloneDeep` on the `req` object could potentially be pretty costly and a real performance hit